test/hca_metadata_api/test.py

```diff
@@ -754,6 +778,57 @@ def test_name_substitution(self):
             self.assertEqual(expected_slash_after, with_slash_after)
 
 
+class TestSchema(AzulUnitTestCase):
+
+    def json_body(self, domain: str) -> JSON:
+        return {
+            'content': {
+                'describedBy': f'https://{domain}/type/protocol/7.1.0/protocol',
+                'protocol_core': {
+                    'protocol_id': 'test-protocol'
+                },
+                'schema_type': 'protocol'
+            },
+            'hca_ingest': {
+                'document_id': str(uuid4()),
+                'submissionDate': '2024-01-01T00:00:01.000Z',
+            }
+        }
+
+    def test_valid_schema_domain(self):
+        json_body = self.json_body(domain='schema.humancellatlas.org')
+        file = Protocol.from_json(json_body)
```

Comment:
Why is the variable called `file` when it's referring to an instance of `Protocol`?

Reply:
The variable name was a oversight from an earlier version that was using a `File` schema